src/mapgen_valleys.cpp

```diff
@@ -238,7 +238,8 @@ void MapgenValleys::makeChunk(BlockMakeData *data)
 	// Generate noise maps and base terrain height.
 	// Modify heat and humidity maps.
 	calculateNoise();
-
+std::mt19937 gen;
+	gen();
```

Comment:
Interesting .. why is this added to only this mapgen? I assume it sets up a pseudorandom generator but i can't see where `gen` is used in the code.